framework/include/utils/BilinearInterpolation.h

```diff
@@ -11,18 +11,21 @@
 
 // MOOSE includes
 #include "ColumnMajorMatrix.h"
+#include "BidimensionalInterpolation.h"
 
 // C++ includes
 #include <vector>
 
+class BidimensionalInterpolation;
+
```

Comment:
```suggestion\
```

Reply:
uh?

Reply:
lol. Remove the forward declaration is what I was after

Reply:
ok done